packages/vc-verification/test/vc-issuer-verification.ts

```diff
@@ -341,7 +342,8 @@ function testSuite() {
 
   describe('Issuer verification', () => {
     it('verifies issuer, where the role is issued by did', async () => {
-      let ipfsCID = await didStore.save(JSON.stringify(adminVC));
+      const adminJWT = new JWT(adminKeys);
+      let ipfsCID = await didStore.save(await adminJWT.sign(adminVC));
```

Comment:
Could you clarify what is the purpose of additional signing?

Reply:
This would align with the way `OffChainClaim` are published as they are signed before being published.

Reply:
I think we should change our implementation instead of forcing user to create unnecessary signature